static/src/javascripts/projects/commercial/modules/hosted/video.js

```diff
@@ -116,7 +116,8 @@ define([
 
                         player.ready(function () {
                             var vol;
-                            duration = parseInt(this.duration(), 10);
+                            var player = this;
```

Comment:
I don't think this variable is needed as we're assigning it [above](https://github.com/guardian/frontend/pull/14486/files#diff-547d3fcebab67a2a4a843b1d1c287f52R113).